modules/standard/FileSystem.chpl

```diff
@@ -1205,12 +1205,9 @@ iter listdir(path: string = ".", hidden: bool = false, dirs: bool = true,
     return chpl_rt_direntptr_getname(this);
   }
 
-  var dir: DIRptr;
-  var ent: direntptr;
-  var err:syserr = ENOERR;
-  dir = opendir(unescape(path).c_str());
+  var dir: DIRptr = opendir(unescape(path).c_str());
   if (!is_c_nil(dir)) {
-    ent = readdir(dir);
+    var ent: direntptr = readdir(dir);
```

Comment:
Are these changes just to tidy the code or are the required for correct functioning?

Reply:
Just to tidy the code. I assume they do not affect functioning.